src/components-examples/material/checkbox/checkbox-overview/checkbox-overview-example.css

```diff
@@ -1 +1,12 @@
-/** No CSS for this example */
+.example-section {
+  margin: 10px 0;
```

Comment:
IIRC the Material spec was either in 4px or 8px increments so maybe we should change these margins to be 12px and the one below to be 4px?

Reply:
Updated